src/vs/workbench/parts/files/electron-browser/fileActions.ts

```diff
@@ -633,9 +633,8 @@ class BaseDeleteFileAction extends BaseFileAction {
 
 			// Confirm for moving to trash
 			else if (this.useTrash) {
-				const message = distinctElements.length > 1 ? getConfirmMessage(nls.localize('confirmMoveTrashMessageMultiple', "Are you sure you want to delete the following {0} files?", distinctElements.length), distinctElements.map(e => e.resource))
-					: distinctElements[0].isDirectory ? nls.localize('confirmMoveTrashMessageFolder', "Are you sure you want to delete '{0}' and its contents?", distinctElements[0].name)
-						: nls.localize('confirmMoveTrashMessageFile', "Are you sure you want to delete '{0}'?", distinctElements[0].name);
+				const message = this.getMoveToTrashMessage(distinctElements);
```

Comment:
@pradeepmurugesan it seems to me the same issue exists for the code below (after the comment "Confirm for deleting permanently") when bypassing the trash. Can we enrich your existing method `getMoveToTrashMessage` to also handle the case of deleting without trash? Then maybe it should be one method `getDeleteMessage` that has a parameter for wether the trash is used or not.